apps/calendar/js/app.js

```diff
@@ -1,5 +1,6 @@
 define(function(require, exports) {
 'use strict';
+/* global IntlHelper */
```

Comment:
I'm pretty sure there is. I'm not familiar with rjs, so it was hard for me to write it this way.

Additionally, `mozIntl` is a shim on HTML Intl API, so I feel like it's in a grey zone between being a library and a public API. Most of it will eventually end up in Intl API, so thinking of using it the same way like window.Intl is intuitive for me.
As for `IntlHelper`, it is our custom code, but it benefits from being a singleton because it makes it possible to cache a single instance of Intl formatter between many uses on one document. I don't know how would it translate to rjs, but if you can help me update the patch to match it, I'll be happy to follow!